Fingerprint_pipeline/make_ccle_29k_coding_snps_step2.R

```diff
@@ -0,0 +1,46 @@
+library(tidyverse)
+# These steps are required for lifting over the map to hg38
```

Comment:
could you push a python version of this instead?

How should I call it? what are the inputs? what are the outputs?

Reply:
can we push this multi step pipeline into one part?